<?xml version="1.0"?>
<testsuites>
<testsuite name="Root Suite" timestamp="2022-04-25T19:04:27" tests="0" file="cypress/integration/secondTest.spec.js" time="0.0000" failures="0"></testsuite>
<testsuite name="Test log out" timestamp="2022-04-25T19:04:27" tests="1" time="3.2110" failures="0">
  
  <testcase name="Test log out verify use can log out successfully" time="3.2110" classname="verify use can log out successfully"></testcase>
  
</testsuite>

<testsuite name="Root Suite" timestamp="2022-04-25T19:05:09" tests="0" file="cypress/integration/testingD/firstTestConduit.spec.js" time="0.0000" failures="0"></testsuite>
<testsuite name="Test with backend" timestamp="2022-04-25T19:05:10" tests="3" time="6.2390" failures="0">
  
  <testcase name="Test with backend should gave tags with routing object " time="3.2640" classname="should gave tags with routing object "></testcase>
  
  <testcase name="Test with backend Verify the local feed likes count" time="2.9750" classname="Verify the local feed likes count"></testcase>
  
</testsuite>

<testsuite name="Root Suite" timestamp="2022-04-25T19:05:24" tests="0" file="cypress/integration/testingD/refactorServerRoute.spec.js" time="0.0000" failures="0"></testsuite>
<testsuite name="Test with backend / refactoring use intercept" timestamp="2022-04-25T19:05:24" tests="5" time="14.8970" failures="2">
  
  <testcase name="Test with backend / refactoring use intercept verify correct request and response" time="7.6720" classname="verify correct request and response">
    
    <failure message="expected 422 to equal 200" type="AssertionError"><![CDATA[AssertionError: expected 422 to equal 200
    at Context.eval (http://localhost:4200/__cypress/tests?p=cypress\integration\testingD\refactorServerRoute.spec.js:124:42)]]></failure>
    
  </testcase>
  
  <testcase name="Test with backend / refactoring use intercept should gave tags with routing object " time="1.7650" classname="should gave tags with routing object "></testcase>
  
  <testcase name="Test with backend / refactoring use intercept Verify the local feed likes count" time="2.5780" classname="Verify the local feed likes count"></testcase>
  
  <testcase name="Test with backend / refactoring use intercept Delete new article  in global feed" time="2.8820" classname="Delete new article  in global feed">
    
    <failure message="`cy.request()` failed on:

https://api.realworld.io/api/articles

The response we received from your web server was:

  &gt; 422: Unprocessable Entity

This was considered a failure because the status code was not `2xx` or `3xx`.

If you do not want status codes to cause failures pass the option: `failOnStatusCode: false`

-----------------------------------------------------------

The request we sent was:

Method: POST
URL: https://api.realworld.io/api/articles
Headers: {
  &quot;Connection&quot;: &quot;keep-alive&quot;,
  &quot;Authorization&quot;: &quot;Token eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImN5dGVzdEB0ZXN0LmNvbSIsInVzZXJuYW1lIjoiQ3lUZXN0IiwiaWF0IjoxNjUwOTEzNTYwLCJleHAiOjE2NTYwOTc1NjB9.PL2Sn4W1c8brvPj5ZpQUioYrhvhiAhFV9hTUFCFtqZE&quot;,
  &quot;user-agent&quot;: &quot;Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Cypress/9.5.3 Chrome/94.0.4606.81 Electron/15.3.5 Safari/537.36&quot;,
  &quot;accept&quot;: &quot;*/*&quot;,
  &quot;accept-encoding&quot;: &quot;gzip, deflate&quot;,
  &quot;content-type&quot;: &quot;application/json&quot;,
  &quot;content-length&quot;: 124
}
Body: {&quot;article&quot;:{&quot;tagList&quot;:[],&quot;title&quot;:&quot;Request from API (Postman)&quot;,&quot;description&quot;:&quot;Api testing is easy&quot;,&quot;body&quot;:&quot;angular is cool&quot;}}

-----------------------------------------------------------

The response we got was:

Status: 422 - Unprocessable Entity
Headers: {
  &quot;date&quot;: &quot;Mon, 25 Apr 2022 19:06:01 GMT&quot;,
  &quot;content-type&quot;: &quot;application/json; charset=utf-8&quot;,
  &quot;content-length&quot;: &quot;39&quot;,
  &quot;connection&quot;: &quot;keep-alive&quot;,
  &quot;x-powered-by&quot;: &quot;Express&quot;,
  &quot;access-control-allow-origin&quot;: &quot;*&quot;,
  &quot;etag&quot;: &quot;W/\&quot;27-BNeuaWQ7DHvW/r+pl+qdJiF06Is\&quot;&quot;,
  &quot;via&quot;: &quot;1.1 vegur&quot;,
  &quot;cf-cache-status&quot;: &quot;DYNAMIC&quot;,
  &quot;expect-ct&quot;: &quot;max-age=604800, report-uri=\&quot;https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct\&quot;&quot;,
  &quot;report-to&quot;: &quot;{\&quot;endpoints\&quot;:[{\&quot;url\&quot;:\&quot;https:\\/\\/a.nel.cloudflare.com\\/report\\/v3?s=o1OdX0%2BJqOSP4UBgjwmLs2%2Be0jBXhnIt5wXCfJ7qnOzlY4iUXLDH4fL2pMCwsSDsBKy8q3XSUbIlCqk9yHP%2Bsfi%2BJnQD19TgG0i92gPYtsPUnbZTljbE2nViaD8mt8fZBhJT\&quot;}],\&quot;group\&quot;:\&quot;cf-nel\&quot;,\&quot;max_age\&quot;:604800}&quot;,
  &quot;nel&quot;: &quot;{\&quot;success_fraction\&quot;:0,\&quot;report_to\&quot;:\&quot;cf-nel\&quot;,\&quot;max_age\&quot;:604800}&quot;,
  &quot;server&quot;: &quot;cloudflare&quot;,
  &quot;cf-ray&quot;: &quot;70195a7e4e1a8c78-EWR&quot;,
  &quot;alt-svc&quot;: &quot;h3=\&quot;:443\&quot;; ma=86400, h3-29=\&quot;:443\&quot;; ma=86400&quot;
}
Body: {
  &quot;errors&quot;: {
    &quot;title&quot;: [
      &quot;must be unique&quot;
    ]
  }
}


https://on.cypress.io/request" type="CypressError"><![CDATA[CypressError: `cy.request()` failed on:

https://api.realworld.io/api/articles

The response we received from your web server was:

  > 422: Unprocessable Entity

This was considered a failure because the status code was not `2xx` or `3xx`.

If you do not want status codes to cause failures pass the option: `failOnStatusCode: false`

-----------------------------------------------------------

The request we sent was:

Method: POST
URL: https://api.realworld.io/api/articles
Headers: {
  "Connection": "keep-alive",
  "Authorization": "Token eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImN5dGVzdEB0ZXN0LmNvbSIsInVzZXJuYW1lIjoiQ3lUZXN0IiwiaWF0IjoxNjUwOTEzNTYwLCJleHAiOjE2NTYwOTc1NjB9.PL2Sn4W1c8brvPj5ZpQUioYrhvhiAhFV9hTUFCFtqZE",
  "user-agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Cypress/9.5.3 Chrome/94.0.4606.81 Electron/15.3.5 Safari/537.36",
  "accept": "*/*",
  "accept-encoding": "gzip, deflate",
  "content-type": "application/json",
  "content-length": 124
}
Body: {"article":{"tagList":[],"title":"Request from API (Postman)","description":"Api testing is easy","body":"angular is cool"}}

-----------------------------------------------------------

The response we got was:

Status: 422 - Unprocessable Entity
Headers: {
  "date": "Mon, 25 Apr 2022 19:06:01 GMT",
  "content-type": "application/json; charset=utf-8",
  "content-length": "39",
  "connection": "keep-alive",
  "x-powered-by": "Express",
  "access-control-allow-origin": "*",
  "etag": "W/\"27-BNeuaWQ7DHvW/r+pl+qdJiF06Is\"",
  "via": "1.1 vegur",
  "cf-cache-status": "DYNAMIC",
  "expect-ct": "max-age=604800, report-uri=\"https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct\"",
  "report-to": "{\"endpoints\":[{\"url\":\"https:\\/\\/a.nel.cloudflare.com\\/report\\/v3?s=o1OdX0%2BJqOSP4UBgjwmLs2%2Be0jBXhnIt5wXCfJ7qnOzlY4iUXLDH4fL2pMCwsSDsBKy8q3XSUbIlCqk9yHP%2Bsfi%2BJnQD19TgG0i92gPYtsPUnbZTljbE2nViaD8mt8fZBhJT\"}],\"group\":\"cf-nel\",\"max_age\":604800}",
  "nel": "{\"success_fraction\":0,\"report_to\":\"cf-nel\",\"max_age\":604800}",
  "server": "cloudflare",
  "cf-ray": "70195a7e4e1a8c78-EWR",
  "alt-svc": "h3=\":443\"; ma=86400, h3-29=\":443\"; ma=86400"
}
Body: {
  "errors": {
    "title": [
      "must be unique"
    ]
  }
}


https://on.cypress.io/request
    at http://localhost:4200/__cypress/runner/cypress_runner.js:160659:80
    at tryCatcher (http://localhost:4200/__cypress/runner/cypress_runner.js:13022:23)
    at Promise._settlePromiseFromHandler (http://localhost:4200/__cypress/runner/cypress_runner.js:10957:31)
    at Promise._settlePromise (http://localhost:4200/__cypress/runner/cypress_runner.js:11014:18)
    at Promise._settlePromise0 (http://localhost:4200/__cypress/runner/cypress_runner.js:11059:10)
    at Promise._settlePromises (http://localhost:4200/__cypress/runner/cypress_runner.js:11139:18)
    at _drainQueueStep (http://localhost:4200/__cypress/runner/cypress_runner.js:7729:12)
    at _drainQueue (http://localhost:4200/__cypress/runner/cypress_runner.js:7722:9)
    at Async.../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:4200/__cypress/runner/cypress_runner.js:7738:5)
    at Async.drainQueues (http://localhost:4200/__cypress/runner/cypress_runner.js:7608:14)
From Your Spec Code:
    at Context.eval (http://localhost:4200/__cypress/tests?p=cypress\integration\testingD\refactorServerRoute.spec.js:200:10)]]></failure>
    
  </testcase>
  
</testsuite>

<testsuite name="Root Suite" timestamp="2022-04-25T19:03:28" tests="0" file="cypress/integration/firstTest.spec.js" time="0.0000" failures="0"></testsuite>
<testsuite name="Test with backend" timestamp="2022-04-25T19:03:28" tests="5" time="16.1970" failures="4">
  
  <testcase name="Test with backend verify correct request and response" time="4.4610" classname="verify correct request and response">
    
    <failure message="expected 422 to equal 200" type="AssertionError"><![CDATA[AssertionError: expected 422 to equal 200
    at Context.eval (http://localhost:4200/__cypress/tests?p=cypress\integration\firstTest.spec.js:120:42)]]></failure>
    
  </testcase>
  
  <testcase name="Test with backend intercepting and modifying the request and response" time="4.9170" classname="intercepting and modifying the request and response">
    
    <failure message="Cannot read properties of undefined (reading &apos;description&apos;)" type="TypeError"><![CDATA[TypeError: Cannot read properties of undefined (reading 'description')
    at eval (http://localhost:4200/__cypress/tests?p=cypress\integration\firstTest.spec.js:131:33)]]></failure>
    
  </testcase>
  
  <testcase name="Test with backend should gave tags with routing object" time="2.0390" classname="should gave tags with routing object"></testcase>
  
  <testcase name="Test with backend verify global feed likes count" time="2.5480" classname="verify global feed likes count">
    
    <failure message="expected &apos;&lt;button.btn.btn-sm.btn-outline-primary&gt;&apos; to contain &apos;1&apos;" type="AssertionError"><![CDATA[AssertionError: expected '<button.btn.btn-sm.btn-outline-primary>' to contain '1'
    at Context.eval (http://localhost:4200/__cypress/tests?p=cypress\integration\firstTest.spec.js:161:35)]]></failure>
    
  </testcase>
  
  <testcase name="Test with backend delete a new article in a global feed" time="2.2320" classname="delete a new article in a global feed">
    
    <failure message="`cy.request()` failed on:

https://conduit.productionready.io/api/articles/

The response we received from your web server was:

  &gt; 422: Unprocessable Entity

This was considered a failure because the status code was not `2xx` or `3xx`.

If you do not want status codes to cause failures pass the option: `failOnStatusCode: false`

-----------------------------------------------------------

The request we sent was:

Method: POST
URL: https://conduit.productionready.io/api/articles/
Headers: {
  &quot;Connection&quot;: &quot;keep-alive&quot;,
  &quot;Authorization&quot;: &quot;Token eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImN5dGVzdEB0ZXN0LmNvbSIsInVzZXJuYW1lIjoiQ3lUZXN0IiwiaWF0IjoxNjUwOTEzNDU0LCJleHAiOjE2NTYwOTc0NTR9.bCERqT2xvMb0a7_DU-i-aq3_J-7g8_In4IwfMK5Ra5Y&quot;,
  &quot;user-agent&quot;: &quot;Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Cypress/9.5.3 Chrome/94.0.4606.81 Electron/15.3.5 Safari/537.36&quot;,
  &quot;accept&quot;: &quot;*/*&quot;,
  &quot;accept-encoding&quot;: &quot;gzip, deflate&quot;,
  &quot;content-type&quot;: &quot;application/json&quot;,
  &quot;content-length&quot;: 114,
  &quot;referer&quot;: &quot;https://conduit.productionready.io/api/articles/&quot;
}
Body: {&quot;article&quot;:{&quot;tagList&quot;:[],&quot;title&quot;:&quot;Request from API&quot;,&quot;description&quot;:&quot;API testing is easy&quot;,&quot;body&quot;:&quot;Angular is cool&quot;}}
Redirects: [
  &quot;307: https://api.realworld.io/api/articles/&quot;
]

-----------------------------------------------------------

The response we got was:

Status: 422 - Unprocessable Entity
Headers: {
  &quot;date&quot;: &quot;Mon, 25 Apr 2022 19:04:15 GMT&quot;,
  &quot;content-type&quot;: &quot;application/json; charset=utf-8&quot;,
  &quot;content-length&quot;: &quot;39&quot;,
  &quot;connection&quot;: &quot;keep-alive&quot;,
  &quot;x-powered-by&quot;: &quot;Express&quot;,
  &quot;access-control-allow-origin&quot;: &quot;*&quot;,
  &quot;etag&quot;: &quot;W/\&quot;27-BNeuaWQ7DHvW/r+pl+qdJiF06Is\&quot;&quot;,
  &quot;via&quot;: &quot;1.1 vegur&quot;,
  &quot;cf-cache-status&quot;: &quot;DYNAMIC&quot;,
  &quot;expect-ct&quot;: &quot;max-age=604800, report-uri=\&quot;https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct\&quot;&quot;,
  &quot;report-to&quot;: &quot;{\&quot;endpoints\&quot;:[{\&quot;url\&quot;:\&quot;https:\\/\\/a.nel.cloudflare.com\\/report\\/v3?s=BBUN%2BTn3RQfenuHSySMra96p0meQzdXfbsxCvos2nDcTP1K1ajTajyPmwqgp2nvfcCOzgNDX8F5qj2GUTQaX5slSwzj2iJTpTMVMvchclc9VvieHCl9R5LOChv3B%2F47ytqdl\&quot;}],\&quot;group\&quot;:\&quot;cf-nel\&quot;,\&quot;max_age\&quot;:604800}&quot;,
  &quot;nel&quot;: &quot;{\&quot;success_fraction\&quot;:0,\&quot;report_to\&quot;:\&quot;cf-nel\&quot;,\&quot;max_age\&quot;:604800}&quot;,
  &quot;server&quot;: &quot;cloudflare&quot;,
  &quot;cf-ray&quot;: &quot;701957e8dfa98c11-EWR&quot;,
  &quot;alt-svc&quot;: &quot;h3=\&quot;:443\&quot;; ma=86400, h3-29=\&quot;:443\&quot;; ma=86400&quot;
}
Body: {
  &quot;errors&quot;: {
    &quot;title&quot;: [
      &quot;must be unique&quot;
    ]
  }
}


https://on.cypress.io/request" type="CypressError"><![CDATA[CypressError: `cy.request()` failed on:

https://conduit.productionready.io/api/articles/

The response we received from your web server was:

  > 422: Unprocessable Entity

This was considered a failure because the status code was not `2xx` or `3xx`.

If you do not want status codes to cause failures pass the option: `failOnStatusCode: false`

-----------------------------------------------------------

The request we sent was:

Method: POST
URL: https://conduit.productionready.io/api/articles/
Headers: {
  "Connection": "keep-alive",
  "Authorization": "Token eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImN5dGVzdEB0ZXN0LmNvbSIsInVzZXJuYW1lIjoiQ3lUZXN0IiwiaWF0IjoxNjUwOTEzNDU0LCJleHAiOjE2NTYwOTc0NTR9.bCERqT2xvMb0a7_DU-i-aq3_J-7g8_In4IwfMK5Ra5Y",
  "user-agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Cypress/9.5.3 Chrome/94.0.4606.81 Electron/15.3.5 Safari/537.36",
  "accept": "*/*",
  "accept-encoding": "gzip, deflate",
  "content-type": "application/json",
  "content-length": 114,
  "referer": "https://conduit.productionready.io/api/articles/"
}
Body: {"article":{"tagList":[],"title":"Request from API","description":"API testing is easy","body":"Angular is cool"}}
Redirects: [
  "307: https://api.realworld.io/api/articles/"
]

-----------------------------------------------------------

The response we got was:

Status: 422 - Unprocessable Entity
Headers: {
  "date": "Mon, 25 Apr 2022 19:04:15 GMT",
  "content-type": "application/json; charset=utf-8",
  "content-length": "39",
  "connection": "keep-alive",
  "x-powered-by": "Express",
  "access-control-allow-origin": "*",
  "etag": "W/\"27-BNeuaWQ7DHvW/r+pl+qdJiF06Is\"",
  "via": "1.1 vegur",
  "cf-cache-status": "DYNAMIC",
  "expect-ct": "max-age=604800, report-uri=\"https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct\"",
  "report-to": "{\"endpoints\":[{\"url\":\"https:\\/\\/a.nel.cloudflare.com\\/report\\/v3?s=BBUN%2BTn3RQfenuHSySMra96p0meQzdXfbsxCvos2nDcTP1K1ajTajyPmwqgp2nvfcCOzgNDX8F5qj2GUTQaX5slSwzj2iJTpTMVMvchclc9VvieHCl9R5LOChv3B%2F47ytqdl\"}],\"group\":\"cf-nel\",\"max_age\":604800}",
  "nel": "{\"success_fraction\":0,\"report_to\":\"cf-nel\",\"max_age\":604800}",
  "server": "cloudflare",
  "cf-ray": "701957e8dfa98c11-EWR",
  "alt-svc": "h3=\":443\"; ma=86400, h3-29=\":443\"; ma=86400"
}
Body: {
  "errors": {
    "title": [
      "must be unique"
    ]
  }
}


https://on.cypress.io/request
    at http://localhost:4200/__cypress/runner/cypress_runner.js:160659:80
    at tryCatcher (http://localhost:4200/__cypress/runner/cypress_runner.js:13022:23)
    at Promise._settlePromiseFromHandler (http://localhost:4200/__cypress/runner/cypress_runner.js:10957:31)
    at Promise._settlePromise (http://localhost:4200/__cypress/runner/cypress_runner.js:11014:18)
    at Promise._settlePromise0 (http://localhost:4200/__cypress/runner/cypress_runner.js:11059:10)
    at Promise._settlePromises (http://localhost:4200/__cypress/runner/cypress_runner.js:11139:18)
    at _drainQueueStep (http://localhost:4200/__cypress/runner/cypress_runner.js:7729:12)
    at _drainQueue (http://localhost:4200/__cypress/runner/cypress_runner.js:7722:9)
    at Async.../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:4200/__cypress/runner/cypress_runner.js:7738:5)
    at Async.drainQueues (http://localhost:4200/__cypress/runner/cypress_runner.js:7608:14)
From Your Spec Code:
    at Context.eval (http://localhost:4200/__cypress/tests?p=cypress\integration\firstTest.spec.js:180:10)]]></failure>
    
  </testcase>
  
</testsuite>

<testsuite name="Root Suite" timestamp="2022-04-25T19:04:35" tests="0" file="cypress/integration/testingD/configSettingEnvironment.spec.js" time="0.0000" failures="0"></testsuite>
<testsuite name="Test with backend / refactoring use intercept" timestamp="2022-04-25T19:04:35" tests="5" time="10.9340" failures="2">
  
  <testcase name="Test with backend / refactoring use intercept verify correct request and response" time="5.7590" classname="verify correct request and response">
    
    <failure message="expected 422 to equal 200" type="AssertionError"><![CDATA[AssertionError: expected 422 to equal 200
    at Context.eval (http://localhost:4200/__cypress/tests?p=cypress\integration\testingD\configSettingEnvironment.spec.js:124:42)]]></failure>
    
  </testcase>
  
  <testcase name="Test with backend / refactoring use intercept should gave tags with routing object " time="1.2140" classname="should gave tags with routing object "></testcase>
  
  <testcase name="Test with backend / refactoring use intercept Verify the local feed likes count" time="1.9040" classname="Verify the local feed likes count"></testcase>
  
  <testcase name="Test with backend / refactoring use intercept Delete new article  in global feed" time="2.0570" classname="Delete new article  in global feed">
    
    <failure message="`cy.request()` failed on:

https://api.realworld.io/api/articles

The response we received from your web server was:

  &gt; 422: Unprocessable Entity

This was considered a failure because the status code was not `2xx` or `3xx`.

If you do not want status codes to cause failures pass the option: `failOnStatusCode: false`

-----------------------------------------------------------

The request we sent was:

Method: POST
URL: https://api.realworld.io/api/articles
Headers: {
  &quot;Connection&quot;: &quot;keep-alive&quot;,
  &quot;Authorization&quot;: &quot;Token eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImN5dGVzdEB0ZXN0LmNvbSIsInVzZXJuYW1lIjoiQ3lUZXN0IiwiaWF0IjoxNjUwOTEzNTAxLCJleHAiOjE2NTYwOTc1MDF9.hq-BmyokRuxYVK8u4tMy85N9zOevC8cuwPluYDBX6ZQ&quot;,
  &quot;user-agent&quot;: &quot;Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Cypress/9.5.3 Chrome/94.0.4606.81 Electron/15.3.5 Safari/537.36&quot;,
  &quot;accept&quot;: &quot;*/*&quot;,
  &quot;accept-encoding&quot;: &quot;gzip, deflate&quot;,
  &quot;content-type&quot;: &quot;application/json&quot;,
  &quot;content-length&quot;: 124
}
Body: {&quot;article&quot;:{&quot;tagList&quot;:[],&quot;title&quot;:&quot;Request from API (Postman)&quot;,&quot;description&quot;:&quot;Api testing is easy&quot;,&quot;body&quot;:&quot;angular is cool&quot;}}

-----------------------------------------------------------

The response we got was:

Status: 422 - Unprocessable Entity
Headers: {
  &quot;date&quot;: &quot;Mon, 25 Apr 2022 19:05:02 GMT&quot;,
  &quot;content-type&quot;: &quot;application/json; charset=utf-8&quot;,
  &quot;content-length&quot;: &quot;39&quot;,
  &quot;connection&quot;: &quot;keep-alive&quot;,
  &quot;x-powered-by&quot;: &quot;Express&quot;,
  &quot;access-control-allow-origin&quot;: &quot;*&quot;,
  &quot;etag&quot;: &quot;W/\&quot;27-BNeuaWQ7DHvW/r+pl+qdJiF06Is\&quot;&quot;,
  &quot;via&quot;: &quot;1.1 vegur&quot;,
  &quot;cf-cache-status&quot;: &quot;DYNAMIC&quot;,
  &quot;expect-ct&quot;: &quot;max-age=604800, report-uri=\&quot;https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct\&quot;&quot;,
  &quot;report-to&quot;: &quot;{\&quot;endpoints\&quot;:[{\&quot;url\&quot;:\&quot;https:\\/\\/a.nel.cloudflare.com\\/report\\/v3?s=3qZ7vqU8bAWdnkGWPwKQUUXv7oq0SfELxd32%2BXo7Zs%2BvpGDNs2iG97bS%2FamWlaAHcBytN6Q5dEHdpSoH3132%2FQBfr0vLrudGwgl%2BGaq%2F%2FsjdKXdLWzNVTbsE155ROcZiIYIf\&quot;}],\&quot;group\&quot;:\&quot;cf-nel\&quot;,\&quot;max_age\&quot;:604800}&quot;,
  &quot;nel&quot;: &quot;{\&quot;success_fraction\&quot;:0,\&quot;report_to\&quot;:\&quot;cf-nel\&quot;,\&quot;max_age\&quot;:604800}&quot;,
  &quot;server&quot;: &quot;cloudflare&quot;,
  &quot;cf-ray&quot;: &quot;7019590bcbd38c78-EWR&quot;,
  &quot;alt-svc&quot;: &quot;h3=\&quot;:443\&quot;; ma=86400, h3-29=\&quot;:443\&quot;; ma=86400&quot;
}
Body: {
  &quot;errors&quot;: {
    &quot;title&quot;: [
      &quot;must be unique&quot;
    ]
  }
}


https://on.cypress.io/request" type="CypressError"><![CDATA[CypressError: `cy.request()` failed on:

https://api.realworld.io/api/articles

The response we received from your web server was:

  > 422: Unprocessable Entity

This was considered a failure because the status code was not `2xx` or `3xx`.

If you do not want status codes to cause failures pass the option: `failOnStatusCode: false`

-----------------------------------------------------------

The request we sent was:

Method: POST
URL: https://api.realworld.io/api/articles
Headers: {
  "Connection": "keep-alive",
  "Authorization": "Token eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImN5dGVzdEB0ZXN0LmNvbSIsInVzZXJuYW1lIjoiQ3lUZXN0IiwiaWF0IjoxNjUwOTEzNTAxLCJleHAiOjE2NTYwOTc1MDF9.hq-BmyokRuxYVK8u4tMy85N9zOevC8cuwPluYDBX6ZQ",
  "user-agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Cypress/9.5.3 Chrome/94.0.4606.81 Electron/15.3.5 Safari/537.36",
  "accept": "*/*",
  "accept-encoding": "gzip, deflate",
  "content-type": "application/json",
  "content-length": 124
}
Body: {"article":{"tagList":[],"title":"Request from API (Postman)","description":"Api testing is easy","body":"angular is cool"}}

-----------------------------------------------------------

The response we got was:

Status: 422 - Unprocessable Entity
Headers: {
  "date": "Mon, 25 Apr 2022 19:05:02 GMT",
  "content-type": "application/json; charset=utf-8",
  "content-length": "39",
  "connection": "keep-alive",
  "x-powered-by": "Express",
  "access-control-allow-origin": "*",
  "etag": "W/\"27-BNeuaWQ7DHvW/r+pl+qdJiF06Is\"",
  "via": "1.1 vegur",
  "cf-cache-status": "DYNAMIC",
  "expect-ct": "max-age=604800, report-uri=\"https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct\"",
  "report-to": "{\"endpoints\":[{\"url\":\"https:\\/\\/a.nel.cloudflare.com\\/report\\/v3?s=3qZ7vqU8bAWdnkGWPwKQUUXv7oq0SfELxd32%2BXo7Zs%2BvpGDNs2iG97bS%2FamWlaAHcBytN6Q5dEHdpSoH3132%2FQBfr0vLrudGwgl%2BGaq%2F%2FsjdKXdLWzNVTbsE155ROcZiIYIf\"}],\"group\":\"cf-nel\",\"max_age\":604800}",
  "nel": "{\"success_fraction\":0,\"report_to\":\"cf-nel\",\"max_age\":604800}",
  "server": "cloudflare",
  "cf-ray": "7019590bcbd38c78-EWR",
  "alt-svc": "h3=\":443\"; ma=86400, h3-29=\":443\"; ma=86400"
}
Body: {
  "errors": {
    "title": [
      "must be unique"
    ]
  }
}


https://on.cypress.io/request
    at http://localhost:4200/__cypress/runner/cypress_runner.js:160659:80
    at tryCatcher (http://localhost:4200/__cypress/runner/cypress_runner.js:13022:23)
    at Promise._settlePromiseFromHandler (http://localhost:4200/__cypress/runner/cypress_runner.js:10957:31)
    at Promise._settlePromise (http://localhost:4200/__cypress/runner/cypress_runner.js:11014:18)
    at Promise._settlePromise0 (http://localhost:4200/__cypress/runner/cypress_runner.js:11059:10)
    at Promise._settlePromises (http://localhost:4200/__cypress/runner/cypress_runner.js:11139:18)
    at _drainQueueStep (http://localhost:4200/__cypress/runner/cypress_runner.js:7729:12)
    at _drainQueue (http://localhost:4200/__cypress/runner/cypress_runner.js:7722:9)
    at Async.../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:4200/__cypress/runner/cypress_runner.js:7738:5)
    at Async.drainQueues (http://localhost:4200/__cypress/runner/cypress_runner.js:7608:14)
From Your Spec Code:
    at Context.eval (http://localhost:4200/__cypress/tests?p=cypress\integration\testingD\configSettingEnvironment.spec.js:200:10)]]></failure>
    
  </testcase>
  
</testsuite>

</testsuites>